packages/apex-parser-ast/docs/bugs/docSymbols/bug-fixes-analysis.md

```diff
@@ -0,0 +1,196 @@
+# LSP Document Symbol Quality Improvements Analysis
```

Comment:
As talked about with Shane, do we really want to commit this to the repo? It might be nice to have for context in the PR description but as for something to keep for good? Don't think we need it.